src/main/java/org/opensrp/web/controller/UniqueIdController.java

```diff
@@ -113,6 +117,22 @@
 			logger.error("", e);
 			message = "Sorry, an error occured when generating the qr code pdf";
 		}
+		finally {
```

Comment:
can we use try with resources... https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Yes, updated PR with try with resources.